src/pretix/presale/templates/pretixpresale/event/base.html

```diff
@@ -224,6 +224,15 @@ <h1>
     {% if request.event.settings.privacy_url %}
         <li><a href="{% safelink request.event.settings.privacy_url %}" target="_blank" rel="noopener">{% trans "Privacy policy" %}</a></li>
     {% endif %}
+    {% if request.event.settings.accessibility_url %}
+        <li><a href="{% safelink request.event.settings.accessibility_url %}" target="_blank" rel="noopener">
+            {{ request.event.settings.accessibility_title }}
```

Comment:
What if I leave the title blank? Should we fallback to a default value or shouldn’t we link to it? IMHO we should not link to it, if there is not title.